tests/test_views.py

```diff
@@ -382,28 +382,26 @@ def test_env_theme(theme, client):
            f'//td{xpath_cls("list-buttons-column")}/form/button/@title': ['Delete record']},
-    'de': {f'//ul{xpath_cls("nav navbar-nav")}/li/a/text()': ['Start', 'Bookmarks', 'Tags', 'Statistic'],
-           f'//ul{xpath_cls("nav nav-tabs")}/li/a/text()': ['Liste (1)', 'Erstellen', 'Random', 'Filter hinzufügen', '10 Elemente'],
+    'de': {f'//ul{xpath_cls("nav navbar-nav")}/li/a/text()': ['Start', 'Lesezeichen', 'Schilder', 'Statistik'],
+           f'//ul{xpath_cls("nav nav-tabs")}/li/a/text()': ['Liste (1)', 'Erstellen', 'Zufälliger', 'Filter hinzufügen', '10 Elemente'],
            f'//td{xpath_cls("list-buttons-column")}/a/@title': ['Eintrag ansehen', 'Eintrag bearbeiten'],
-           f'//td{xpath_cls("list-buttons-column")}/form/button/@title': ['Delete record']},
-    'fr': {f'//ul{xpath_cls("nav navbar-nav")}/li/a/text()': ['Accueil', 'Bookmarks', 'Tags', 'Statistic'],
-           f'//ul{xpath_cls("nav nav-tabs")}/li/a/text()': ['Liste (1)', 'Créer', 'Random', 'Ajouter un filtre', '10 items'],
+           f'//td{xpath_cls("list-buttons-column")}/form/button/@title': ['Delete record']},  # ['Datensatz löschen']},
+    'fr': {f'//ul{xpath_cls("nav navbar-nav")}/li/a/text()': ['Accueil', 'Signets', 'Étiquettes', 'Statistique'],
+           f'//ul{xpath_cls("nav nav-tabs")}/li/a/text()': ['Liste (1)', 'Créer', 'Aléatoire', 'Ajouter un filtre', '10 items'],
            f'//td{xpath_cls("list-buttons-column")}/a/@title': ['Afficher L\'enregistrement', 'Modifier enregistrement'],
-           f'//td{xpath_cls("list-buttons-column")}/form/button/@title': ['Delete record']},
-    'ru': {f'//ul{xpath_cls("nav navbar-nav")}/li/a/text()': ['Главная', 'Bookmarks', 'Tags', 'Statistic'],
-           f'//ul{xpath_cls("nav nav-tabs")}/li/a/text()': ['Список (1)', 'Создать', 'Random', 'Добавить Фильтр', '10 элементы'],
+           f'//td{xpath_cls("list-buttons-column")}/form/button/@title': ['Delete record']},  # ['Supprimer l\'enregistrement']},
+    'ru': {f'//ul{xpath_cls("nav navbar-nav")}/li/a/text()': ['Главная', 'Закладки', 'Теги', 'Статистика'],
+           f'//ul{xpath_cls("nav nav-tabs")}/li/a/text()': ['Список (1)', 'Создать', 'Случайная', 'Добавить Фильтр', '10 элементы'],
            f'//td{xpath_cls("list-buttons-column")}/a/@title': ['Просмотр записи', 'Редактировать запись'],
-           f'//td{xpath_cls("list-buttons-column")}/form/button/@title': ['Delete record']},
+           f'//td{xpath_cls("list-buttons-column")}/form/button/@title': ['Delete record']},  # ['Удалить запись']},
 }
+locale = env_fixture('BUKUSERVER_LOCALE', params=['en', 'de', 'fr', 'ru', None])
 
 @pytest.mark.gui
 @pytest.mark.slow
-@pytest.mark.parametrize('locale', ['en', 'de', 'fr', 'ru', None])
-def test_env_locale(bukudb, app, client, locale):
+def test_env_locale(bukudb, locale, client):
     strings = _DICT[locale or 'en']
     _add_rec(bukudb, 'http://example.com')
-    if locale:
-        app.config.update({'BUKUSERVER_LOCALE': locale})
```

Comment:
Turns out, this approach doesn't actually work with app translations (as opposed to Flask-Admin ones)